layouts/partials/navbar.html

```diff
@@ -20,7 +21,7 @@
 			</li>
 			{{ end }}
 			<!-- HARD CODED DROP DOWN FOR RELEASE SELECTION -->
-			<li class="nav-item dropdown">
+            <li class="nav-item dropdown" {{ if not ($versions) }} style="display: none" {{ end }}>
```

Comment:
wrap the whole `li` element in the conditional rather than visually hide the content

```
{{ if not ($versions) }} 
<li class="nav-item dropdown">
  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
  Releases
  </a>
  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
    <a class="dropdown-item" href="/pct/changelog">PCT</a>
    <a class="dropdown-item" href="/prm/changelog">PRM</a>
  </div>
</li>
{{ end }}
```